src/java.base/share/classes/jdk/internal/math/DoubleConsts.java

```diff
@@ -73,12 +73,20 @@ private DoubleConsts() {}
      */
     public static final long    SIGNIF_BIT_MASK = 0x000FFFFFFFFFFFFFL;
 
+    /**
+     * Bit mask to isolate the magnitude bits (combined exponent and
+     * significand fields) of a {@code double}.
+     */
+    public static final long    MAG_BIT_MASK = 0x7FFFFFFFFFFFFFFFL;
```

Comment:
Might be more "obviously correct" if these new fields were initialized as ~SIGN_BIT_MASK.